scripts/create-a-m-o/index.js

```diff
@@ -0,0 +1,136 @@
+const outdent = require('outdent');
+const chalk = require('chalk');
+const readline = require('readline');
+
+const {
+  getAMOType,
+  prepareName,
+  createFiles,
+} = require('./functions.js');
+
+process.stdin.setEncoding('utf8');
+
+console.log(chalk.cyan(outdent`
+    Hello Dear developer, thank you for contributing with us. 😊
```

Comment:
"thank you for your contribution" somehow sounds better. Not sure if "thank you for contributing with us" isn't correct english. But I know that this is just a copypasta.

Reply:
thank you for your contribution represents finished code, contributing instead is present continous. Cause he just created the new components, i think is better use the present continues rather than past

Reply:
What I mean is the "WITH US" part, not the rest. Is he contributing WITH or TO us? Or probably "to our project"

To make a complete sentence: "Thank you for your contribution to this project."
This is correct english. Not sure if "Thank you for contributing WITH us" is.

But this is a detail. Just wanted to point it out, I don't know myself if that might still be legit, do write whatever you want here. :)